client/my-sites/post-type-list/empty-content.jsx

```diff
@@ -36,7 +35,6 @@ function PostTypeListEmptyContent( {
 				title={ title }
 				action={ action }
 				actionURL={ editPath }
-				actionHoverCallback={ preloadEditor }
```

Comment:
We've removed the only usage of `actionHoverCallback`, maybe we should remove the prop altogether from `EmptyContent` ?